src/Documentation/sidebar.json

```diff
@@ -120,6 +120,80 @@
         "label": "Managing External Data",
         "slug": "managing-external-data"
       },
+      {
+        "label": "Remotes",
+        "slug": "remotes",
+        "source": "remotes/index.md",
+        "children": [
+          {
+            "label": "Local",
+            "slug": "local"
+          },
+          {
+            "label": "HTTP",
+            "slug": "http"
+          },
+          {
+            "label": "SSH",
+            "slug": "ssh"
+          },
+          {
+            "label": "Amazon S3",
+            "slug": "amazon"
+          },
+          {
+            "label": "S3 Compatible",
+            "slug": "s3"
+          },
+          {
+            "label": "Azure",
+            "slug": "azure"
+          },
+          {
+            "label": "Google Cloud Storage",
+            "slug": "gs"
+          },
+          {
+            "label": "HDFS",
+            "slug": "hdfs"
+          },
+          {
+            "label": "Aliyun OSS",
+            "slug": "oss"
+          }
+        ]
+      },
+      {
+        "label": "External Data",
+        "slug": "external-data",
+        "source": "external-data/index.md",
+        "children": [
+          {
+            "label": "Local Filesystem",
```

Comment:
Local should go very last in all cases. SSH,  S3, Google Cloud, Azure, HTTP, HDFS - something like this is better

Reply:
Local supports more cases than the other types, and it also can be used for the mounted data storages (NAS etc.) So, it seems like an important one. Why should it go last?